frontend/src/index.css

```
@@ -755,14 +746,93 @@ textarea {
+  grid-template-rows: repeat(3, var(--aurral-dot-loader-tile-size));
+  gap: var(--aurral-dot-loader-gap);
+  flex: 0 0 auto;
+  vertical-align: middle;
+  color: var(--aurral-text-muted);
+}
+
+.aurral-dot-loader__dot {
+  width: var(--aurral-dot-loader-tile-size);
+  height: var(--aurral-dot-loader-tile-size);
+  border-radius: 50%;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)